sdk/search/azure-search-documents/CHANGELOG.md

```diff
@@ -1,14 +1,14 @@
 # Release History
 
-## 11.6.0-beta.4 (Unreleased)
+## 11.6.0-beta.4 (2023-01-13)
```

Comment:
Ideally, preview releases for a library that has a GA version should have a section for "Features Added". Do we have any new features for this preview release?